font-test-data/test_data/ttx/embedded_bitmaps.ttx

```diff
@@ -0,0 +1,313 @@
+        <colorRef value="0"/>
+        <startGlyphIndex value="1"/>
+        <endGlyphIndex value="2"/>
+        <ppemX value="7"/>
+        <ppemY value="7"/>
+        <bitDepth value="1"/>
+        <flags value="1"/>
+      </bitmapSizeTable>
+      <!-- GlyphIds are written but not read. The firstGlyphIndex and
+           lastGlyphIndex values will be recalculated by the compiler. -->
+      <eblc_index_sub_table_3 imageFormat="2" firstGlyphIndex="1" lastGlyphIndex="2">
+        <glyphLoc id="1" name="g1"/>
+        <glyphLoc id="2" name="g2"/>
+      </eblc_index_sub_table_3>
+    </strike>
+    <strike index="1">
+      <bitmapSizeTable>
+        <sbitLineMetrics direction="hori">
+          <ascender value="12"/>
+          <descender value="5"/>
+          <widthMax value="9"/>
+          <caretSlopeNumerator value="0"/>
+          <caretSlopeDenominator value="0"/>
+          <caretOffset value="0"/>
+          <minOriginSB value="0"/>
+          <minAdvanceSB value="0"/>
+          <maxBeforeBL value="12"/>
+          <minAfterBL value="-5"/>
+          <pad1 value="0"/>
+          <pad2 value="0"/>
+        </sbitLineMetrics>
+        <sbitLineMetrics direction="vert">
+          <ascender value="12"/>
+          <descender value="5"/>
+          <widthMax value="0"/>
+          <caretSlopeNumerator value="0"/>
+          <caretSlopeDenominator value="0"/>
+          <caretOffset value="0"/>
+          <minOriginSB value="0"/>
+          <minAdvanceSB value="0"/>
+          <maxBeforeBL value="0"/>
+          <minAfterBL value="0"/>
+          <pad1 value="0"/>
+          <pad2 value="0"/>
+        </sbitLineMetrics>
+        <colorRef value="0"/>
+        <startGlyphIndex value="3"/>
+        <endGlyphIndex value="3"/>
+        <ppemX value="15"/>
+        <ppemY value="15"/>
+        <bitDepth value="1"/>
+        <flags value="1"/>
+      </bitmapSizeTable>
+      <!-- GlyphIds are written but not read. The firstGlyphIndex and
+           lastGlyphIndex values will be recalculated by the compiler. -->
+      <eblc_index_sub_table_2 imageFormat="5" firstGlyphIndex="3" lastGlyphIndex="3">
+        <imageSize value="20"/>
+        <BigGlyphMetrics>
+          <height value="17"/>
+          <width value="9"/>
+          <horiBearingX value="0"/>
+          <horiBearingY value="12"/>
+          <horiAdvance value="9"/>
+          <vertBearingX value="-4"/>
+          <vertBearingY value="-9"/>
+          <vertAdvance value="0"/>
+        </BigGlyphMetrics>
+        <glyphLoc id="3" name="g3"/>
+      </eblc_index_sub_table_2>
+    </strike>    
+  </EBLC>
+
+  <CBDT>
+    <header version="3.0"/>
+    <strikedata index="0">
+      <cbdt_bitmap_format_17 name="numbersign">
+        <SmallGlyphMetrics>
+          <height value="128"/>
+          <width value="136"/>
+          <BearingX value="0"/>
+          <BearingY value="101"/>
+          <Advance value="136"/>
+        </SmallGlyphMetrics>
+        <rawimagedata>
```

Comment:
is this a real image? Does it need to be so long or could it be a much shorter fragment of example data?

Reply:
It is real. I thought having a real PNG would make sense but now realize that it doesn’t matter. I can shorten this.

Reply:
I was worried that ttx might validate the PNG but it appears not. Truncated to the first 8 bytes.